client/src/components/Register/MethodTile.scss

```diff
@@ -83,17 +87,28 @@
   &__unavailable {
     &-mask {
       background: $tile-mask-background;
-      height: 100%;
+      border-top-left-radius: $tile-border-radius;
+      border-top-right-radius: $tile-border-radius;
+      color: $tile-mask-colour;
       width: 100%;
       left: 0;
-      padding: $tile-padding;
+      padding: $tile-padding/2 $tile-padding;
       position: absolute;
       top: 0;
+      z-index: 1;
```

Comment:
We can avoid the `z-index` by moving the `-mask` after the content.

Reply:
Yeah, probably.
This is a less code is better code situation, so I'll indulge :)

Reply:
I did this.

Reply:
z-index's usually end up being a hassle somewhere along the line. Best practice is to avoid it if possible (with DOM order).